aggregator/src/aggregator/aggregation_job_writer.rs

```diff
@@ -120,10 +120,12 @@ where
     pub fn put(
         &mut self,
         aggregation_job: AggregationJob<SEED_SIZE, Q, A>,
-        report_aggregations: Vec<RA>,
+        mut report_aggregations: Vec<RA>,
     ) -> Result<(), Error> {
         self.update_aggregation_parameter(aggregation_job.aggregation_parameter());
 
+        report_aggregations.sort_unstable_by_key(RA::ord);
```

Comment:
I think this sort may not be necessary since we're just using order-preserving rayon operations, but I haven't checked all the docs closely yet. `par_bridge()`/`ParallelBridge` does not preserve the order, but others can, since they start with full knowledge of the items to be processed, and subdivide both the input iterator and output destination.

Reply:
This is needed (at least) because aggregation job continuation's preprocessing step to match prep steps to report aggs will now place any failed report aggs at the front of the list, so they will no longer be in `ord` order.

Reply:
We definitely need this sort now that data is flowing back over a channel.